phpcs-ruleset.xml

```diff
@@ -2,4 +2,10 @@
 <ruleset name="Behat for WordPress coding standards">
   <rule ref="PSR1" />
   <rule ref="PSR2" />
+  <rule ref="PSR2.Methods.FunctionCallSignature.SpaceAfterCloseBracket">
```

Comment:
In terms of my initial desired code standards (always welcome to discuss and change these with you, of course), the best list I'm aiming for so far is explained on bottom of https://github.com/paulgibbs/behat-wordpress-extension/blob/master/CONTRIBUTING.md